source/settings.rst

```diff
@@ -1095,7 +1097,91 @@ This page contains settings for the Git Extensions :ref:`commit` dialog. Note th
   .. setting:: Vertical ruler position
 
     Position for ruler in TextEditor controls. Set to 0 to disable.
-    (This should be moved to the TextEditor context menu.)
+
+.. settingsgroup:: Diff appearance
+
+  Control how diffs are presented. This is set in the diff viewer context menu.
+
+    .. image:: /images/settings/diff-appearance.png
+
+  .. setting:: Patch view
+
+    The patch view shows the changes in a file as a patch. This is the default view.
+
+    The appearance can to some extent be controlled by Git Settings, see also :ref:`settings-diff-coloring`.
+
+  .. setting:: Git word diff
+
+    The appearance uses Git word diff as described in `GitDiff#--word-diff <https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---word-diffltmodegt>`_
+
+    This is not a "patch" view, and so cannot be used to apply line patches.
+    It is intended as a contrast to the default view why Git default view is changed to show a minimal diff.
```

Comment:
I think that this sentence should be rewritten...

Reply:
Read it again and again and I still don't get the sense of this sentence 😕

Is `It is intended, as a contrast to the default view, to show a minimal diff.` not good enough?

Reply:
Yes, that is better, will change.